test/framework/integration_framework/iroha_instance.cpp

```diff
@@ -52,11 +52,21 @@ namespace integration_framework {
         irohad_log_manager_(std::move(irohad_log_manager)),
         log_(std::move(log)) {}
 
+  void IrohaInstance::init() {
+    auto init_result = instance_->init();
+    if (auto error =
+            boost::get<iroha::expected::Error<std::string>>(&init_result)) {
+      std::string error_msg("Irohad startup failed: ");
+      error_msg.append(error->error);
+      log_->critical(error_msg);
```

Comment:
Suggest `log_->critical("Irohad startup failed: {}", error->error);`

Reply:
I would like to avoid duplication, so I reuse the message. But you are right about the necessity to use `{}`-formatting.